indico/modules/events/layout/templates/layout_meeting_lecture.html

```diff
@@ -1,10 +1,23 @@
 {% extends 'layout/base.html' %}
-{% from 'forms/_form.html' import simple_form %}
+{% from 'forms/_form.html' import simple_form, form_rows %}
 
 {% block title %}
     {%- trans %}Layout{% endtrans -%}
 {% endblock %}
 
 {%- block content %}
-    {{ simple_form(form, save_reminder=true, back_button=false) }}
+     {% call simple_form(form, save_reminder=true, back_button=false) %}
+         <div>
+            {{ form_rows(form, fields=('timetable_theme', 'timetable_by_room', 'timetable_detailed')) }}
```

Comment:
the only field this form has is `timetable_theme`, and since it has only one field just keep the simple `form_rows(form)` call without specifying fields